src/main/java/command/CommandAddTaskHandler.java

```diff
@@ -0,0 +1,11 @@
+package command;
+import main.Main;
```

Comment:
```suggestion
package command;

import main.Main;
```
Should this have a line separation in between package and import? Notice this in a few other areas.